biodatasets/osiris/osiris.py

```diff
@@ -0,0 +1,302 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from typing import List
+from numpy import int32
+import xml.etree.ElementTree as ET
+import itertools
+import uuid
+
+import datasets
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@ARTICLE{Furlong2008,
+  author = {Laura I Furlong and Holger Dach and Martin Hofmann-Apitius and Ferran Sanz},
+  title = {OSIRISv1.2: a named entity recognition system for sequence variants
+  of genes in biomedical literature.},
+  journal = {BMC Bioinformatics},
+  year = {2008},
+  volume = {9},
+  pages = {84},
+  doi = {10.1186/1471-2105-9-84},
+  pii = {1471-2105-9-84},
+  pmid = {18251998},
+  timestamp = {2013.01.15},
+  url = {http://dx.doi.org/10.1186/1471-2105-9-84}
+}
+"""
+
+_DATASETNAME = "osiris"
+
+_DESCRIPTION = """\
+The OSIRIS corpus is a set of MEDLINE abstracts manually annotated
+with human variation mentions. The corpus is distributed under the terms
+of the Creative Commons Attribution License
+Creative Commons Attribution 3.0 Unported License,
+which permits unrestricted use, distribution, and reproduction in any medium,
+provided the original work is properly cited (Furlong et al, BMC Bioinformatics 2008, 9:84).
+"""
+
+_HOMEPAGE = "https://github.com/rockt/SETH/tree/master/resources/OSIRIS"
```

Comment:
Could you please change this to : https://sites.google.com/site/laurafurlongweb/databases-and-tools/corpora/
This is the official webpage, though the download link is broken

Reply:
Done.